package.json

```diff
@@ -40,6 +40,7 @@
     "babel": "^5.8.25",
     "babel-core": "^5.8.25",
     "babel-eslint": "^4.1.1",
+    "babel-relay-plugin": "0.7.1",
```

Comment:
Im not able to repro passing tests without having babel-relay-plugin in devdeps or deps.  What do you mean `scripts`?